azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/collection/ResourcesInGroup.java

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Let's use the shorter license header in each file since this long one is already in LICENSE.txt.